### DISCUSSION TITLE
Release Schedule?

### DISCUSSION THREAD

**Discussion by grahamplace** [Snowflake-Labs/terraform-provider-snowflake#287] [Q&A] [answered]
Hey all! I'm wondering when we can expect a new release to be cut? 

We want to start using the Network Policy resource added in https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/271 — it's of course possible to build the provider ourselves but I'd prefer not to update our build system which installs from the official source

Thanks!

**Comment by ryanking** [comment, marked as answer]
Hi @grahamplace, we don't have a regular release schedule, but probably should. I can go ahead and do a release now to get that out.

**Reply by ryanking** [reply]
[0.18.0](https://github.com/chanzuckerberg/terraform-provider-snowflake/releases/tag/v0.18.0) is available now.
